CollAction/Frontend/app/global/MapMarkerItem.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import renderComponentIf from "./renderComponentIf";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faMapMarkerAlt } from "@fortawesome/free-solid-svg-icons";
+
+library.add(faMapMarkerAlt);
+
+class MapMarkerItem extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
```

Comment:
Is this necessary if the implementation is empty?

Reply:
Both the constructor and componentDidMount are not needed here. Less code is beter.

Reply:
fixed in d0cb67b